api/client/gro_client.py

```diff
@@ -280,9 +280,9 @@ def find_data_series(self, **kwargs):
 
         See https://developers.gro-intelligence.com/data-series-definition.html
 
-        This method uses :meth:`~.search` to find entities by name and :meth:`~.get_data_series` to
-        find available data series for all possible combinations of the entities, and
-        :meth:`~.rank_series_by_source`.
+        This method uses :meth:`~.search`, :meth:`~.get_data_series`,
+        :meth:`~.get_available_timeandfrequency` and  :meth:`~.rank_series_by_source`.
```

Comment:
function name should be `get_available_timefrequency` ?

Reply:
done